examples/gallery/plot/points-categorical.py

```diff
@@ -0,0 +1,69 @@
+"""
+Color points by categories
+---------------------------
+The :meth:`pygmt.Figure.plot` method can be used to plot symbols which are
+color-coded by categories. In the example below, we show how the
+`palmerpenguins dataset <https://github.com/allisonhorst/palmerpenguins>`__
+can be visualized. Here, we can pass the individual categories included in
+the species column directly to the ``color`` parameter via
+``color=df.species.cat.codes.astype(int)``. Additionally, we have to set
+``cmap=True``. A desired colormap can be selected via the :meth:`pygmt.makecpt`
+method.
+"""
+
+import numpy as np
+import pandas as pd
+import pygmt
+
+# Load sample penguins data and convert 'species' column to categorical dtype
+df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/penguins.csv")
+df["species"] = df.species.astype(dtype="category")
+
+# Use pygmt.info to get region bounds (xmin, xmax, ymin, ymax)
+# The below example will return a numpy array like [30.0, 60.0, 12.0, 22.0]
+region = pygmt.info(
+    table=df[["bill_length_mm", "bill_depth_mm"]],  # x and y columns
+    per_column=True,  # report output as a numpy array
+    spacing="3/2",  # rounds x and y intervals by 3 and 2 respectively
+)
+
+# Make our 2D categorical scatter plot, coloring each of the 3 species differently
+fig = pygmt.Figure()
+
+# Generate basemap of 10cm x 10cm size
+fig.basemap(
+    region=region,
+    projection="X10c/10c",
+    frame=[
+        'xafg+l"Bill length in mm"',
```

Comment:
```suggestion
        'xafg+l"Bill length (mm)"',
```

Reply:
We usually put the units in (), right?

Reply:
😄 Same comments as mine (https://github.com/GenericMappingTools/pygmt/pull/1006#discussion_r590592975)

Reply:
I always used *unit in XX* or *unit/XX* but I'm fine with the changes if all other examples follow the same convention.